.gitmodules

```diff
@@ -22,3 +22,15 @@
 	path = mfe/frontend-app-home
 	url = git@github.com:Abstract-Tech/community-theme-home.git
 	branch = community-theme
+[submodule "mfe/frontend-component-footer"]
```

Comment:
@snglth Since we customized the course-authoring, authn and discussions MFE i think we should add them as submodules too